hat/assets/js/apps/Iaso/domains/dataSources/components/VersionsDialog.js

```diff
@@ -269,7 +272,7 @@ const VersionsDialog = ({ renderTrigger, source }) => {
             <Table
                 data={sortedData}
                 columns={tableColumns(source)}
-                redirectTo={() => {}}
+                redirectTo={() => { }}
```

Comment:
you can remove this line, Table as a default props set to `redirectTo: () => null,`